sa/sa.go

```diff
@@ -885,11 +885,13 @@ func (ssa *SQLStorageAuthority) CountCertificatesRange(ctx context.Context, star
 func (ssa *SQLStorageAuthority) CountPendingAuthorizations(ctx context.Context, regID int64) (count int, err error) {
 	err = ssa.dbMap.SelectOne(&count,
 		`SELECT count(1) FROM pendingAuthorizations
-		 WHERE registrationID = :regID AND
-				expires > :now`,
+     WHERE registrationID = :regID AND
```

Comment:
Can you match the indent here with the `SELECT` above it? That would seem to better match the other instances of the multi-line SQL statements elsewhere.